getting-started-gradle/pom.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Just above the out/ binary files are added. They shouldn't be. BTW, shouldn't they be in build/ as far as I understand the Gradle build?

Reply:
Totally right